R/estimate_KM.R

```diff
@@ -72,13 +72,14 @@ estimate_KM <- function(
 ){
 
 # Capture input to validate user input for data argument -----------------
+  dots <- rlang::dots_list(...)
+  estimate_KM_args <- as.list(environment())
```

Comment:
I would only keep what we actually need from the environment() call. In clinical trials we work with big data so we don't want to keep information that is not required eg estimate_KM_args[["data"]]. If the strata is NULL is should be defaulted to "Overall" with label "Overall"

Reply:
I'm not sure if I see the point of this additional line since we know the strata and we can remove whatever is in front of 'xx = ' easily + look for the corresponding label in the dataframe. Perhaps another argument in survfit object that contains strata labels might be more useful?

Reply:
What I was expecting here is that you are capturing the actual environment of 'data' so that we don't get in trouble with wrapper functions.

Reply:
now just exporting the strata variables as a named list that include the labels (if they are present). Add the env, I feel, should be addressed in another PR.